### DISCUSSION TITLE
Allow tag icon to have text visible over recordings

### DISCUSSION THREAD

**Discussion by izzygold** [blakeblackshear/frigate#13381] [Ideas]
Hi,
On review page, for each thumbnail there is small icon on the top left corner, I would be very happy if I had a knob that enables me to have text next to the icon.

the icon is very clear for People detection, but the only way to know from the audio detection is to hover over the icon.

I have hundreds of these detections and I'd really like to be able to scroll over them and immediately see which noise happened.

Thanks!

**Comment by NickM-27** [comment]
You can hover to view the text. It will also be possible in 0.15 to open a more info menu

**Reply by izzygold** [reply]
Hi,
I am not sure why your closed this. 

As I mentioned in my message - you can definitely hover to view the text, but I would be happy if it could be right there without having to hover over it. The motivation is that we have the same icon for different audio events, so I do think its worth while to be more explicit about what event happened.

**Reply by izzygold** [reply]
also - I don't want to click anything in order to get this info so I don't think that a more info menu would help in that matter.

**Reply by NickM-27** [reply]
right and as I said it has already been implemented in 0.15 where there will be a more info menu that shows this. Having text on the main image would cover too much of the image.

**Reply by NickM-27** [reply]
it is possible in the future that more icons can be added for specific audio types, but the UI is working as it was designed and text isn't compatible with that

**Reply by izzygold** [reply]
How about allowing the user to add custom icons?
or as an alternative - If I get someone to create new icons for the events I care about, could they be pushed? what format should they be in?

**Reply by NickM-27** [reply]
We have an icon pack already, it's just a matter of linking the label to the icon

**Reply by izzygold** [reply]
cool, do you think theres any chance to see icons like civil defense siren and explosions on 0.15?

**Reply by NickM-27** [reply]
I don't see why not
